src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  margin: 2px 6px 2px 2px;
+  width: 12px;
+  height: 12px;
+  background-image: url(.\images\star-active.svg);
+  background-size: cover;
+}
+
+.stars--0 > :nth-child(n+1) {
+  background-image: url(.\images\star.svg);
+}
+
+.stars--1 > :nth-child(n+2) {
+  background-image: url(.\images\star.svg);
+}
+
+.stars--2 > :nth-child(n+3) {
+  background-image: url(.\images\star.svg);
+}
+
+.stars--3 > :nth-child(n+4) {
+  background-image: url(.\images\star.svg);
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url(.\images\star.svg);
+}
+
+.stars--5 > :nth-child(n+6) {
+  background-image: url(.\images\star.svg);
```

Comment:
Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)